bionic/deriver.py

```diff
@@ -117,12 +117,10 @@ def should_collapse_dnode(dnode):
             if _include_detail:
                 return False
 
-            if isinstance(dnode, ast.EntityNode):
+            if dnode.is_entity():
```

Comment:
Wondering about the style: what about doing something like `if dnode.subtype() == ENTITY` ?

Reply:
That would work too, but you'd have to import `Subtype` and then write `if dnode.subtype == Subtype.ENTITY`, which I think is more annoying. (Note that Python doesn't have static imports so if you want to import just `ENTITY`, you need to separately do `ENTITY = Subtype.ENTITY`, and repeat it for each subtype.)

Python doesn't really have a great idiom for doing this kind of pattern matching (until [PEP 622](https://www.python.org/dev/peps/pep-0622/), maybe) so all of the options are a little clunky.